mautic/mautic
# Fix default order direction  in controllers

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
This is follow up of PRs from @alfredoct96  to add columns and change order by in other section (campaigns, segments, pages..)

The order by icon of the column was not displayed or in incorrect order.  

Now

![image](https://user-images.githubusercontent.com/462477/154058265-00f723b2-0a37-4cd9-aa5d-492f27eaa1d4.png)



1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Let's check after login
- assets
- forms
- pages
- campaigns
- sms
- emails

3. The ordering icon should be in the correct column and in correct order
4. Also the PR keep direction If you switch ordering with column




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10862"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
